In [1]:
from cplane_julia import JuliaPlane

In [2]:
myPlane = JuliaPlane(0)

In [3]:
%time myPlane.refresh(0.5j)

CPU times: user 32.3 s, sys: 772 ms, total: 33 s
Wall time: 33.6 s


We computed the time it took to refresh a 2000 by 2000 `JuliaPlane` instance without the numba decorator, resulting in the following time:

`CPU times: user 32.3 s, sys: 772 ms, total: 33 s` <br>
`Wall time: 33.6 s`

In [2]:
myPlane = JuliaPlane(0)

In [3]:
%time myPlane.refresh(0.5j)

CPU times: user 1.14 s, sys: 476 ms, total: 1.62 s
Wall time: 1.67 s


We computed the time it took to refresh a 2000 by 2000 JuliaPlane instance without the numba decorator, resulting in the following time:

`CPU times: user 1.14 s, sys: 476 ms, total: 1.62 s` <br>
`Wall time: 1.67 s`